content/doc/book/pipeline/overview.adoc

```diff
@@ -1,4 +1,297 @@
 ---
-layout: refresh
-refresh_to_post_id: '/doc/book/pipeline/getting-started'
+layout: section
 ---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+
+= Pipeline Overview
```

Comment:
This section and its contents don't fit at all IMHO. There is already the "Getting Started" section which introduces users to creating a first Pipeline, then "The Jenkinsfile" walks the user through the various bits of Pipeline that they should consider after creating their first Pipeline.


The platform-specific examples I think should be excised entirely. There are some other pieces of this document that might make sense to integrate into "The Jenkinsfile" though